srcCpp/perftest_publisher.cxx

```diff
@@ -34,7 +34,7 @@ bool perftest_cpp::_testCompleted_scan = true; // In order to enter into the sca
 const char *perftest_cpp::_LatencyTopicName = "Latency";
 const char *perftest_cpp::_AnnouncementTopicName = "Announcement";
 const char *perftest_cpp::_ThroughputTopicName = "Throughput";
-const DDS_Duration_t timeout_wait_for_ack = {0, 100000000};
+const int timeout_wait_for_ack_nsec = 100000000;
```

Comment:
I would create:
const int timeout_wait_for_ack_sec = 0;
Because you are using it in several places. If you modify, you will do only in one place.
And the same in the rest of languages.

Reply:
Done